### ISSUE TITLE
Red on light red background does not provide high enough contrast

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6574]
Error notification Red on Light Red background does not provide high enough contrast. Does not meet the contrast minimum of 4.5:1 for text.

![not enough contrast](https://user-images.githubusercontent.com/964912/50982926-20975180-14b3-11e9-94b7-1967c66fad18.png)

**Comment by outofambit** [comment]
cc @ampinsk

**Comment by agisilaos** [comment]
Hey @outofambit. Would love to do an audit and check how's Github desktop accessibility-wise. Check if colors are accessible for a start. Do you have specific areas in the app in mind where you believe colors may be off etc so I can start from there?

**Comment by shiftkey** [comment]
@agisilaos no need to do anything currently about auditing - we've got a report that we're working through to turn into actionable issues like this one

**Comment by outofambit** [comment]
@agisilaos lets chat more after I've got all these issues written up! there definitely may be more to audit!

**Comment by ampinsk** [comment]
I think this one is also fine: 

https://colorable.jxnblk.com/9e1c23/ffdce0

![screen shot 2019-01-10 at 11 15 06 am](https://user-images.githubusercontent.com/10404068/50991111-c0aba580-14c8-11e9-85e5-ede4e9973a85.png)

Unless someone wants to verify, I think we can close this.

**Comment by ampinsk** [comment]
Hey @agisilaos would you like to verify this one?

**Comment by ampinsk** [comment]
I confirmed with the design systems team that passing AA is ✅ so I'm going to close this out.
